azure-cosmos-db/1.0/modules/ROOT/pages/azure-cosmos-db-connector-reference.adoc

```diff
@@ -0,0 +1,1939 @@
+
+[%header,cols="20s,25a,30a,15a,10a"]
+|===
+| Field | Type | Description | Default Value | Required
+| Key a| String | Key for this type. |  | x
+| Value a| String | Value for this type. |  | x
+|===
+
+[[DefaultQueryParam]]
+=== Default Query Param
+
+Query parameters automatically added to every outbound request.
+
+[%header,cols="20s,25a,30a,15a,10a"]
+|===
+| Field | Type | Description | Default Value | Required
+| Key a| String | Key for this type. |  | x
+| Value a| String | Value for this type. |  | x
+|===
+
+[[Tls]]
+=== TLS
+
+Configures TLS to provide secure communications for the Mule app.
+
+[%header,cols="20s,25a,30a,15a,10a"]
+|===
+| Field | Type | Description | Default Value | Required
+| Enabled Protocols a| String | Comma-separated list of protocols enabled for this context. |  |
+| Enabled Cipher Suites a| String | Comma-separated list of cipher suites enabled for this context. |  |
+| Trust Store a| <<TrustStore>> | Configures the TLS truststore. |  |
+| Key Store a| <<KeyStore>> | Configures the TLS keystore. |  |
+| Revocation Check a| * <<StandardRevocationCheck>>
+* <<CustomOcspResponder>>
+* <<CrlFile>> | Configures a revocation checking mechanism. |  |
+|===
+
+[[TrustStore]]
+=== Truststore
+
+Configures the truststore for TLS.
+
+[%header,cols="20s,25a,30a,15a,10a"]
+|===
+| Field | Type | Description | Default Value | Required
+| Path a| String | Path to the truststore. Mule resolves the path relative to the current classpath and file system. |  |
+| Password a| String | Password used to protect the truststore. |  |
+| Type a| String | Type of truststore. |  |
+| Algorithm a| String | Encryption algorithm that the truststore uses. |  |
+| Insecure a| Boolean | If `true`, Mule stops performing certificate validations. Setting this to `true` can make connections vulnerable to attacks. |  |
+|===
+
+[[KeyStore]]
+=== Keystore
+
+Configures the keystore for the TLS protocol. The keystore you generate contains a private key and a public certificate.
+
+[%header,cols="20s,25a,30a,15a,10a"]
+|===
+| Field | Type | Description | Default Value | Required
+| Path a| String | Path to the keystore. Mule resolves the path relative to the current classpath and file system. |  |
+| Type a| String | Type of keystore. |  |
+| Alias a| String | Alias of the key to use when the keystore contains multiple private keys. By default, Mule uses the first key in the file. |  |
+| Key Password a| String | Password used to protect the private key. |  |
+| Password a| String | Password used to protect the keystore. |  |
+| Algorithm a| String | Encryption algorithm that the keystore uses. |  |
+|===
+
+[[StandardRevocationCheck]]
+=== Standard Revocation Check
+
+Configures standard revocation checks for TLS certificates.
+
+[%header,cols="20s,25a,30a,15a,10a"]
+|===
+| Field | Type | Description | Default Value | Required
+| Only End Entities a| Boolean | Which elements to verify in the certificate chain:
+
```

Comment:
From here down, it looks like the lists are missing the + sign that indents the text under the bullet.